mixer/cmd/rpms.go

```diff
@@ -15,6 +15,8 @@
 package cmd
 
 import (
+	"errors"
```

Comment:
is this the `errors` import you wanna use? Other parts of the code aew using `https://github.com/pkg/errors` instead.

Reply:
If you do a grep for \"errors\" vs pkg/errors\" in the repo you'll see we are very inconsistent. I grabbed the one in the standard library because it provided the functionality I need.

Reply:
Indeed. Yet another thing to add to our todo list